cpp/cmake/thirdparty/get_libcudacxx.cmake

```diff
@@ -14,10 +14,13 @@
 
 # This function finds libcudacxx and sets any additional necessary environment variables.
 function(find_and_configure_libcudacxx)
-  include(${rapids-cmake-dir}/cpm/libcudacxx.cmake)
 
-  rapids_cpm_libcudacxx(BUILD_EXPORT_SET raft-exports
-                        INSTALL_EXPORT_SET raft-exports)
+  if(RAFT_ENABLE_cuco_DEPENDENCY)
```

Comment:
Likewise here, we should hoist this out to where we call `find_and_configure_libcudacxx`